src/sezzle-checkout-button/README.md

```diff
@@ -22,12 +30,21 @@ The following are 4 methods for installing the Sezzle checkout button. The `Inst
 The button appearance can now be customized as needed using the below keys. Here is an example of the default configuration:
 
 ```
-    <script>
-    new SezzleCheckoutButton({
-		theme: "light", <!-- accepts "light" and "dark" (based on site background) -->
-        template: "Checkout with %%logo%%"  <!-- accepts "Checkout with %%logo%%", "Pay with %%logo%%", or "%%logo%%" -->
-    }).init();
-    </script>
+  {{ "sezzle-checkout-button.js" | asset_url | script_tag }}
+  <script>
+    const eventLogger = new EventLogger({
+      merchantUUID:"xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+    });
+    try{
+      new SezzleCheckoutButton({
+      theme: "light", <!-- accepts "light" and "dark" (based on site background) -->
+          template: "Checkout with %%logo%%"  <!-- accepts "Checkout with %%logo%%", "Pay with %%logo%%", or "%%logo%%" -->
+      }).init();
+      eventLogger.sendEvent("checkout-button-onload")
```

Comment:
We will need to add these two event types to widget-server as well.

Reply:
Yes, I'm working on WS and SFS changes.